sources/Models/Vector3.cpp

```diff
@@ -0,0 +1,13 @@
+//
+// Created by cbihan on 26/05/2021.
+//
+
+#include "Vector3.hpp"
+
+namespace BBM
+{
+	RAY::Vector3 operator*(const Vector3f &v)
```

Comment:
operator* should take void if you want to use it as *vector3, this one is for arithmetic operator

Reply:
Yes, I was talking about a conversion operator. See [this](https://en.cppreference.com/w/cpp/language/cast_operator).

Reply:
fixed -> https://github.com/AnonymusRaccoon/Bomberman/pull/64/commits/07e2d5bc35aadd0db4cf31ef5a71ac19f06a42c9